src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -2631,6 +2634,8 @@ namespace SimAirServingZones {
 					// Re-evaluate air loop components with new actuated variables
 					++NumCalls;
 					SimAirLoopComponents( AirLoopNum, FirstHVACIteration );
+					// pass inner iteration loop (i.e., SolveWaterCoilController) result back to this loop
+					ControllerConvergedFlag = PrimaryAirSystem( AirLoopNum ).ControlConverged( AirLoopControlNum );
```

Comment:
Hard to pass back convergence of OA sys water coils. This method eased implementation.